ui/components/component-library/avatar-network/avatar-network.types.ts

```diff
@@ -0,0 +1,62 @@
+import { ValidTag } from '../text/text.types';
+import type { BoxProps } from '../../ui/box/box.d';
+import {
+  BackgroundColor,
+  BorderColor,
+  TextColor,
+} from 'ui/helpers/constants/design-system';
+
+export enum AvatarNetworkSize {
+  Xs = 'xs',
+  Sm = 'sm',
+  Md = 'md',
+  Lg = 'lg',
+  Xl = 'xl',
+}
+
+/**
+ * Props for the AvatarNetwork component
+ */
+export interface AvatarNetworkProps extends BoxProps {
```

Comment:
I think we could extend `AvatarBaseProps` here instead of BoxProps and remove `backgroundColor`, `borderColor`,  `color`, `as` types altogether?

Reply:
Yeah, that would be great.